tests/unittests/Foundation/NSStringInternalTests.mm

```diff
@@ -14,6 +14,8 @@
 //
 //******************************************************************************
 
+#ifdef _WIN32
+
```

Comment:
it depends on the internal function _reverseString

---
In reply to: [70350216](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70350216) [](ancestors = 70350216)